recipes/emsarray/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "emsarray" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
```

```suggestion
  name: {{ name|lower }}-meta
```
This is a meta-package which will not actually get built.